speculos/mcu/screen.py

```diff
@@ -171,20 +170,15 @@ def mousePressEvent(self, event: QMouseEvent):
         QApplication.setOverrideCursor(Qt.DragMoveCursor)
 
     def mouseReleaseEvent(self, event: QMouseEvent):
+        self.mouse_offset = event.pos()
         x, y = self._get_x_y()
         if x >= 0 and x < self._width and y >= 0 and y < self._height:
+            # Send a press to update the last pressed x,y to the app
```

Comment:
Can you add a comment explaining that it is mainly to support the "software" swipe where a pressed event with an updated position must be sent before the released event.